### ISSUE TITLE
Creata a nettcpbinding like configuration

### ISSUE THREAD

**Issue by Franki1986** [protobuf-net/protobuf-net.Grpc#37]
Bello,

I am wondering if it is anyhow possible to create a duplex communication with low level TCP Socket. Something like nettcpbinding in wcf.
Maybe also with help of Tcplistener and TCPClient.

**Comment by Franki1986** [comment]
Yes, I am asking for this because I am moving a net framework application with wcf partially to a net core application, but I don't want to get a worse situation tgan before, but if I can to improve every part of my application.

**Comment by mgravell** [comment]
I don't think that's something I can put into *here*, due to all the assumptions that gRPC makes; however, the idea of a low footprint basic message-passing RPC stack: that makes sense, and recently I started doing some exploration of this over in protobuf-net.MessagePipes; it isn't anywhere near ready, though. In that case, the originating idea was named/anonymous pipes, but the transport doesn't matter - just "some stream-like feed"
